src/backend/TrafficCourts/Test/Workflow.Service/Mappings/MappingTests.cs

```diff
@@ -0,0 +1,107 @@
+        Assert.Equal(source.ViolationTicket?.DriversLicenceNumber, target.ViolationTicket.DriversLicenceNumber);
+        Assert.Equal(source.ViolationTicket?.DriversLicenceProvince, target.ViolationTicket.DriversLicenceProvince);
+        Assert.Equal(source.ViolationTicket?.Birthdate.Year, target.ViolationTicket.Birthdate?.Year);
+        Assert.Equal(source.ViolationTicket?.Address, target.ViolationTicket.Address);
+        Assert.Equal(source.ViolationTicket?.City, target.ViolationTicket.City);
+        Assert.Equal(source.ViolationTicket?.Province, target.ViolationTicket.Province);
+        Assert.Equal(source.ViolationTicket?.PostalCode, target.ViolationTicket.PostalCode);
+        Assert.Equal(source.ViolationTicket?.IssuedDate, target.ViolationTicket.IssuedDate);
+        Assert.Equal(source.ViolationTicket?.OrganizationLocation, target.ViolationTicket.OrganizationLocation);
+        Assert.Equal(source.ViolationTicket?.ViolationTicketCounts.Count, target.ViolationTicket.ViolationTicketCounts.Count);
+        List<ViolationTicketCount> ticketCounts = new(target.ViolationTicket.ViolationTicketCounts);
+        for (int i = 0; i < source.ViolationTicket?.ViolationTicketCounts.Count; i++)
+        {
+            Assert.Equal(source.ViolationTicket?.ViolationTicketCounts[i].Count.ToString(), ticketCounts[i].Count.ToString());
+            Assert.Equal(source.ViolationTicket?.ViolationTicketCounts[i].Description, ticketCounts[i].Description);
+            Assert.Equal(source.ViolationTicket?.ViolationTicketCounts[i].FullSection, ticketCounts[i].FullSection);
+            Assert.Equal(source.ViolationTicket?.ViolationTicketCounts[i].ActRegulation, ticketCounts[i].ActRegulation);
+            Assert.Equal(source.ViolationTicket?.ViolationTicketCounts[i].TicketedAmount, ticketCounts[i].TicketedAmount);
+            Assert.Equal(source.ViolationTicket?.ViolationTicketCounts[i].IsAct, ticketCounts[i].IsAct);
+            Assert.Equal(source.ViolationTicket?.ViolationTicketCounts[i].IsRegulation, ticketCounts[i].IsRegulation);
+        }
+
+        Assert.Equal(source.LegalRepresentation?.LawFirmName, target.LegalRepresentation.LawFirmName);
+        Assert.Equal(source.LegalRepresentation?.LawyerFullName, target.LegalRepresentation.LawyerFullName);
+        Assert.Equal(source.LegalRepresentation?.LawyerEmail, target.LegalRepresentation.LawyerEmail);
+        Assert.Equal(source.LegalRepresentation?.LawyerAddress, target.LegalRepresentation.LawyerAddress);
+        Assert.Equal(source.LegalRepresentation?.LawyerPhoneNumber, target.LegalRepresentation.LawyerPhoneNumber);
+
+        Assert.Equal(source.DisputedCounts.Count, target.DisputedCounts.Count);
+        List<DisputedCount> disputedCounts = new(target.DisputedCounts);
+        for (int i = 0; i < source.DisputedCounts?.Count; i++)
+        {
+            Assert.Equal(source.DisputedCounts?[i].Plea.GetTypeCode(), disputedCounts[i].Plea.GetTypeCode());

```

Comment:
I'm not this is correct or how automapper deals with Enums, but this works for now.